addons/stock/views/stock_picking_type_views.xml

```diff
@@ -100,7 +100,7 @@
                             <field attrs='{"invisible": [("code", "not in", ["incoming", "outgoing", "internal"])]}' name="return_picking_type_id" string="Returns Type"/>
                             <field name="default_location_return_id" attrs='{"invisible": [("code", "not in", ["incoming", "outgoing", "internal"])]}' groups="stock.group_stock_multi_locations"/>
                             <field name="create_backorder"/>
-                            <field name="show_operations"/>
+                            <field name="show_operations" attrs='{"invisible": [("code", "not in", ["incoming", "outgoing", "internal"])]}'/>
```

Comment:
I would use the new computed field `show_picking_type` in the domain  https://github.com/odoo/odoo/commit/d4a4a3c1de83ae284aee35085bc19efdf78a9906

Reply:
Okie dokie, we may as well fix the bug in saas-16.4 now that we can! https://github.com/odoo/odoo/pull/131517